### ISSUE TITLE
No way to automatically start playback of next epsiode for TV shows

### ISSUE THREAD

**Issue by coopermf** [jellyfin/jellycon#273]
There is no way to get jellycon to automatically play into the next episode. I thought unchecking the option prompt before play (under events>on playback stop) would cause the addon to start playback of the next episode without needing the user to press anything but it just stops at the end of playback and gives no prompts.

I have tried playing to the end of an episode across multiple devices and have tried using the kodi addon Up Next but haven't gotten anything to play into the next episode.

Here's a link to a portion of the jellyfin log after testing playback
[https://pastebin.com/gYSkCDNX](url)

**System**
 - OS: Android TV 11.0.0 (Shield TV 2019)
 - Jellyfin Version: 10.8.9
 - Kodi Version: 20.0
 - Addon Version: 0.6.1+py3

I would be very grateful if we could get a setting that would automatically start the next episode on finish or integration with an addon like Up Next.

**Comment by mcarlton00** [comment]
There's already integration with the up next addon, and it was improved a bit in #262 in the last release (0.7.0), so I'm not sure what issue it is that you're seeing.  It _should_ be working properly.  You might just need to update, since you're still on 0.6.1 for some reason.

**Comment by Ozymandyaz** [comment]
I am actually having trouble getting UpNext to work with jellycon and I have updated to 0.7.0. (On Nexus BTW...) I enabled debug logging on both Jellycon and UpNext and tailed the log, so I can see JC setting the Next Episode, but then UpNext complains that no next episode was found at the trigger time. Happy to put together a debug log and attach it here or open another issue if it will be helpful.
